src/index.js

```diff
@@ -171,35 +171,44 @@ function EvolvClient(options) {
    * optimization statistics.
    */
   this.confirm = function() {
-    store.isEntryPoint()
-      .then(function(entryPoint) {
-        if (!entryPoint) {
-          return;
-        }
+    waitFor(context, EFFECTIVE_GENOME_UPDATED, function() {
+      const remoteContext = context.remoteContext;
+      const allocations = (remoteContext.experiments || {}).allocations // undefined is a valid state, we want to know if its undefined
```

Comment:
Missing semicolon, also this doesn't seem to be reducing the number of characters, and is resulting in an unused empty object being created, it might be better to move this into the if statement.